packages/components/src/slot-fill/fill.ts

```diff
@@ -11,7 +11,7 @@ import useSlot from './use-slot';
 import type { FillComponentProps } from './types';
 
 export default function Fill( { name, children }: FillComponentProps ) {
-	const { registerFill, unregisterFill } = useContext( SlotFillContext );
+	const registry = useContext( SlotFillContext );
```

Comment:
Is this a stable value?

Reply:
Yes, as long as the context provider is the same, the `registry` is a constant object.